src/backend/gporca/libgpopt/src/operators/CExpressionPreprocessor.cpp

```diff
@@ -214,6 +214,7 @@ CExpressionPreprocessor::PexprSimplifyQuantifiedSubqueries(CMemoryPool *mp,
 		1 == (*pexpr)[0]->DeriveMaxCard().Ull())
 	{
 		CExpression *pexprInner = (*pexpr)[0];
+		GPOS_ASSERT(NULL != pexprInner);
```

Comment:
it looks like we are accessing `(*pexpr)[0]` before the assert.

Reply:
This assert is need for below while cycle

Reply:
109. [here](https://github.com/arenadata/gpdb/blob/fd5939d4f491238bbd28be88b98949cd0e2d6fe0/src/backend/gporca/libgpopt/src/operators/CExpressionPreprocessor.cpp#L220) pexprChild may be NULL [via](https://github.com/arenadata/gpdb/blob/fd5939d4f491238bbd28be88b98949cd0e2d6fe0/src/backend/gporca/libgpopt/src/operators/CExpressionPreprocessor.cpp#L221)